### PR TITLE
Generate Conversation

### PR DESCRIPTION
**Pull request by glennrfisher**
- [x] Generate latest version of Conversation
- [x] Update tests for latest version of Conversation

### CODE REVIEW

File: Source/ConversationV1/Models/CreateWorkspace.swift [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2017
+ * Copyright IBM Corporation 2018
```
Comment (mkistler) [inline Source/ConversationV1/Models/CreateWorkspace.swift]: These copyright date changes are going to be problematic.  We really should not be changing the date when the file has not changed.  I realize that the generator has no way to know this.  We might need to handle this in some kind of post-processing step.

File: Source/ConversationV1/Models/EntityCollection.swift [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2017
+ * Copyright IBM Corporation 2018
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
```
Comment (mkistler) [inline Source/ConversationV1/Models/EntityCollection.swift]: All of the collection models may be affected by the pagination changes that the Conversation team are planning to do.  Hopefully this won't require any work on our part except to regen the service, but its something to keep on your radar.

File: Source/ConversationV1/Models/CreateWorkspace.swift (hunk 1 above)
Comment (glennrfisher) [reply Source/ConversationV1/Models/CreateWorkspace.swift]: Sure, good point. I opened an issue in `arf/planning-sdk-squad` to figure out how we want to handle copyright dates.

We should probably maintain the previous copyright dates...but I do wonder if generation changes anything. In some sense, this entire file _was_ created in 2018. It just happens to be almost exactly the same!

(But I don't think that argument would really hold...)

File: Source/ConversationV1/Models/EntityCollection.swift (hunk 2 above)
Comment (glennrfisher) [reply Source/ConversationV1/Models/EntityCollection.swift]: Thanks for the reminder! Yeah, we should regenerate the service after their updates.

**Pull request merged**